src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -598,7 +598,7 @@ private static OpenApiParameter GetOpenApiQueryParameter(string name, string des
         /// <param name="entityName">Name of the entity.</param>
         /// <param name="dbObject">Database object metadata, indicating entity SourceType</param>
         /// <returns>Collection of OpenAPI OperationTypes and whether they should be created.</returns>
-        private Dictionary<OperationType, bool> GetConfiguredRestOperations(string entityName, DatabaseObject dbObject)
+        private static Dictionary<OperationType, bool> GetConfiguredRestOperations(string entityName, DatabaseObject dbObject, RuntimeEntities entities)
```

Comment:
make sure to update comments above. 
Also, do the input arguments `entityName` and `runtimeEntities` make sense in the context of this function? What object is actually needed here?

Reply:
ya we can remove those and just pass in the entity we want.